scripts/build-non-split.js

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env node
+
+const rewire = require('rewire');
+const defaults = rewire('react-scripts/scripts/build.js');
+let config = defaults.__get__('config');
+
+config.optimization.splitChunks = {
+    cacheGroups: {
+        default: false,
+    },
+};
+
+config.optimization.runtimeChunk = false;
+config.output.filename = 'static/js/form-builder.js';
```

Comment:
using this option allows a react web component to have the correct filename without any shell script work arounds.

Reply:
/cc @jonathanmtran